CodeEditModules/Modules/AppPreferences/src/Model/Source Control/SourceControlPreferences.swift

```diff
@@ -61,12 +61,30 @@ public extension AppPreferences {
                                                                           forKey: .showSourceControlChanges) ?? true
             self.includeUpstreamChanges = try container.decodeIfPresent(Bool.self,
                                                                         forKey: .includeUpstreamChanges) ?? true
-            self.comparisonView = try container.decodeIfPresent(Int.self, forKey: .comparisonView) ?? 0
-            self.controlNavigator = try container.decodeIfPresent(Int.self, forKey: .controlNavigator) ?? 0
+            self.comparisonView = try container.decodeIfPresent(ComparisonView.self,
+                                                                forKey: .comparisonView) ?? .localLeft
+            self.controlNavigator = try container.decodeIfPresent(ControlNavigator.self,
+                                                                  forKey: .controlNavigator) ?? .sortByName
             self.defaultBranchName = try container.decodeIfPresent(String.self, forKey: .defaultBranchName) ?? "main"
         }
     }
 
+    /// The style for comparison View
+    /// - **localLeft**: Local Revision on Left Side
+    /// - **localRight**: Local Revision on Right Side
+    enum ComparisonView: String, Codable {
```

Comment:
make sure to change this too!

Reply:
Do you mean such a thing?
`enum RevisionComparisonLayout: String, Codable {`